# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tschmidt/unreal-ircd/exampleconf.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN Unreal3.2/doc/example.conf Unreal3.2-new/doc/example.conf
--- Unreal3.2/doc/example.conf	2004-10-27 20:45:03.000000000 +0200
+++ Unreal3.2-new/doc/example.conf	2005-01-26 18:29:36.459092216 +0100
@@ -31,8 +31,8 @@
  */
 
 /* FOR *NIX, uncomment the following 2lines: */
-//loadmodule "src/modules/commands.so";
-//loadmodule "src/modules/cloak.so";
+loadmodule "modules/commands.so";
+loadmodule "modules/cloak.so";
 
 /* FOR Windows, uncomment the following 2 lines: */
 //loadmodule "modules/commands.dll";
